scipy/stats/tests/test_distributions.py

```diff
@@ -2381,6 +2381,34 @@ def test_logcdf(self):
                         stats.norm.logsf(np.log(x2-mu)/sigma))
 
 
+# Columns are x, a, b, cdf, sf.
+# The values of cdf and sf were computed using mpmath.
+_beta_cdf_sf = np.array([
+    [1e-20, 3.0, 0.5, 3.1249999999999994e-61, 1.0],
```

Comment:
```suggestion
    [1e-20, 3.0, 0.5, 3.1249999999999995e-61, 1.0],
```
Truncates to 4, but rounds to 5, right?

Reply:
The inputs `3.1249999999999994e-61` and `3.1249999999999995e-61` are the same number:
```
In [13]: 3.1249999999999995e-61                                                                                            
Out[13]: 3.1249999999999994e-61

In [14]: 3.1249999999999994e-61 == 3.1249999999999995e-61                                                                  
Out[14]: True
```

Reply:
I didn't mean that it was of practical concern haha. Just shows that I checked.